ocs_ci/ocs/ui/pvc_ui.py

```diff
@@ -42,6 +43,13 @@ def create_pvc_ui(
         logger.info("Click on 'Create Persistent Volume Claim'")
         self.do_click(self.pvc_loc["pvc_create_button"])
 
+        # Choose With Form if given the choice
+        try:
+            logger.info("Choose With form if given the choice")
+            self.do_click(self.pvc_loc["pvc_create_with_form"])
+        except TimeoutException:
+            logger.info("Proceed to PVC creation")
```

Comment:
Why just the logger in except block?

Reply:
If there's no choice of With Form/With data upload, we just proceed to the next step

Reply:
<img width="582" height="474" alt="Image" src="https://github.com/user-attachments/assets/07cb6b34-e4e4-4ec7-b9d0-0d4b30068b12" />
is there case when dropdown does not appear?

Reply:
Yes, in internal mode